Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -1260,13 +1264,8 @@ final class BuildPlanTests: XCTestCase {
         ])
       #endif
 
-      let linkedFileList: String = try fs.readFileContents(buildPath.appending(components: "exe.product", "Objects.LinkFileList"))
-      XCTAssertEqual(linkedFileList, """
-          \(buildPath.appending(components: "exe.build", "main.c.o"))
-          \(buildPath.appending(components: "extlib.build", "extlib.c.o"))
-          \(buildPath.appending(components: "lib.build", "lib.c.o"))
-
-          """)
+        let buildProduct = try XCTUnwrap(result.productMap["exe"])
+        XCTAssertEqual(Array(buildProduct.objects), [buildPath.appending(components: "exe.build", "main.c.o"), buildPath.appending(components: "extlib.build", "extlib.c.o"), buildPath.appending(components: "lib.build", "lib.c.o")])
```

Comment:
nit: format long lines

Reply:
I missed this one on the first `swiftformat` run and now I guess I can't format anymore?

```
❯ swiftformat Tests/BuildTests/BuildPlanTests.swift                   
Running SwiftFormat...
Reading config file at /Users/neonacho/Projects/public/swift-package-manager/.swiftformat
error: Inconsistent whitespace in multi-line string literal at 4053:1 in /Users/neonacho/Projects/public/swift-package-manager/Tests/BuildTests/BuildPlanTests.swift.
```

Reply:
Did something manual now.